teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
+  def initialize(age, specialization, name)
+    super(age, name)
+    @specialization = specialization
+  end
```

Comment:
- [ ] same here, Kindly extend the parent's (person class) constructor with its default values.
hint:

```ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```

Reply:
Changes have been made as per instructions